src/remotepublish.cmd.js

```diff
@@ -327,6 +327,12 @@ class RemotePublishCommand extends AbstractCommand {
         const merged = other.merge(this.config, this._filter);
         this.log.debug(Array.from(merged.strains.keys()).join(', '));
 
+        // ensure preflight is copied... this.config is preferred it would probably be better
+        // to turn the merge around, but then the filter-logic needs to be changed...
+        if (this.config.preflight) {
+          // eslint-disable-next-line no-underscore-dangle
+          merged._cfg.preflight = this.config.preflight;
```

Comment:
note, I didn't want to _fix_ / _change_ this in helix-shared, as the exact semantics is not clear. i.e. since the merging is meant for strains. e..g what happens if src/dst have different preflight configs? or master has one and _this_ not?